src/main/scala/com/wavesplatform/UtxPool.scala

```diff
@@ -26,21 +29,55 @@ class UtxPool(
                  fs: FunctionalitySettings,
                  utxSettings: UtxSettings) extends ScorexLogging {
 
+  private type Portfolios = Map[Address, Portfolio]
+
   private val transactions = new ConcurrentHashMap[ByteStr, Transaction]
+
   private lazy val knownTransactions = CacheBuilder.newBuilder()
     .maximumSize(utxSettings.maxSize * 2)
     .build[ByteStr, Either[ValidationError, Transaction]]()
 
-  private def removeExpired(currentTs: Long): Unit =
-    transactions.entrySet().removeIf(tx => (currentTs - tx.getValue.timestamp).millis > utxSettings.maxTransactionAge)
+  /**
```

Comment:
```scala
private val txPessimisticPortfolios = new ConcurrentHashMap[ByteStr, Portfolios]
```
I know, but it is hard to name this variable, i did my best. The comments here is for clearing things. 

If someone has an idea about naming, you're welcome :)